Xamarin.Forms.Core/Items/ScrolledEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace Xamarin.Forms.Core.Items
```

Comment:
why a new namespace ? please avoid

Reply:
How should this be handled? Extend `Xamarin.Forms.ScrolledEventArgs` with a new class in the same namespace or add new properties and ctor to it?